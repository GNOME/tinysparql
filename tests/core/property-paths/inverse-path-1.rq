select foaf:name(?a) foaf:name(?b) { ?a ^foaf:knows ?b } order by foaf:name(?a) foaf:name(?b)