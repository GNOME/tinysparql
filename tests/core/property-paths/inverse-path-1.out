"Alice"	"Bob"
"Bob"	"Alice"
"Fred"	"Eve"
